.gitignore

```diff
@@ -1 +1,4 @@
 node_modules/
+dist/
+*.log
+bundle.js
```

Comment:
The bundle is used by GitHub Pages - it's meant to be checked in, and should not be ignored.

Reply:
Got it, thanks for catching. ignore entry removed